Makefile

```diff
@@ -189,26 +189,19 @@ librna/librna.a: $(LIBRNA_OBJ)
 
 
 ################################################################################
-# Mercurial version generation
+# Git version generation
 ################################################################################
 
-ifeq "$(shell ls -d .hg 2>/dev/null)" ".hg"
+BRANCH:=$(shell git rev-parse --abbrev-ref HEAD)
 
-ifneq "$(hg)" ""
-src/version.txt: .hg/dirstate
-	$(HG) log -r . --template '{date|isodate} {node} {tags}' > $@
-	$(HG) id | grep '+' | $(SED) 's/^.\++.*$\/ wd modified/' >> $@
-else
-src/version.txt:
-	basename `pwd` > src/version.txt
-
-endif
+.PHONY : src/version.txt
 
+ifeq ($(BRANCH),master)
+src/version.txt :
+	git log -1 --date=format:"%Y.%m.%d" --format="%ad" >$@
 else
-
-src/version.txt:
-	basename `pwd` > src/version.txt
-
+src/version.txt :
+	git log -1 --date=format:"%Y.%m.%d" --format="%ad"-$(BRANCH) >$@
 endif
 
 src/version.cc: src/version.txt
```

Comment:
for codestyle of the generated src/version.cc, can you please add "  " (two whitespaces) in front of `const ...` for visual indent?